### PR TITLE
DI-289 Add deploy of test environment

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-289

## Description

Add deploy to test into pipeline

## Type of change

Delete not appropriate

- New feature (non-breaking change which adds functionality)

## Testing

Please tick the testing that has been completed

- [ ] Unit tests
- [ ] Component tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Component tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: infrastructure/stacks/development-pipeline/codebuild.tf [hunk 1]
```diff
@@ -149,8 +149,8 @@ resource "aws_codebuild_project" "di_build" {
   }
 }
 
-resource "aws_codebuild_project" "di_deploy" {
-  name           = "${var.project_id}-${var.environment}-deploy-stage"
+resource "aws_codebuild_project" "di_deploy_dev" {
```
Comment (rohith1122) [inline infrastructure/stacks/development-pipeline/codebuild.tf:152]: The ticket says test environment and we are calling as dev here

**Review by rohith1122 — COMMENTED** [review]
I would suggest to call either dev or test. I think we are mixing both here

File: infrastructure/stacks/development-pipeline/codebuild.tf (hunk 1 above)
Comment (ed-sparkes) [reply infrastructure/stacks/development-pipeline/codebuild.tf:152]: it is now and dev and test so i renamed the original to dev to distinguish it from test

File: infrastructure/stacks/development-pipeline/codebuild.tf [hunk 2]
```diff
@@ -175,11 +175,77 @@ resource "aws_codebuild_project" "di_deploy" {
 
     environment_variable {
       name  = "PROFILE"
-      value = var.profile
+      value = "dev"
     }
     environment_variable {
       name  = "ENVIRONMENT"
-      value = var.environment
+      value = "dev"
```
[the remark this answers is not in the document]
Comment (ed-sparkes) [reply infrastructure/stacks/development-pipeline/codebuild.tf:182]: yes because the pipeline now deploys both dev and test!

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Pull request merged**